### PR TITLE
add plyranges to R-bundle-Bioconductor for zUMIs

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/z/zUMIs/zUMIs-2.9.7-foss-2023a-R-4.3.2.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ('SAMtools', '1.18'),
+    ('STAR', '2.7.11a'),
+    ('R', '4.3.2'),
+    ('R-bundle-Bioconductor', '3.18', '-R-%(rver)s'),
+]
+
+postinstallcmds = [
+    "sed -i '33 s/${zumis}/zumis/' %(installdir)s/zUMIs.sh",
+    "sed -i '43,46d' %(installdir)s/zUMIs.sh",
+    "sed -i '7,10d' %(installdir)s/zUMIs.sh",
```
Comment (jfgrimm) [inline easybuild/easyconfigs/z/zUMIs/zUMIs-2.9.7-foss-2023a-R-4.3.2.eb:30]: it would be good to add a quick comment on what these `sed` commands are doing

**Comment by jfgrimm** [comment]
@boegelbot: please test @ jsc-zen3

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/e9d206d547cd8d6bd578db8f3a598b2a for a full test report.

**Comment by jfgrimm** [comment]
Going in, thanks @pavelToman!

**Pull request merged**